### PR TITLE
llama 1.1.1

### PR DESCRIPTION
**Pull request by antonmedv**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/llama.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match "llama", pipe_output(bin/"llama", "--help").strip
```
Comment (chenrui333) [inline Formula/llama.rb:15]:
```suggestion
    # panic: open /dev/tty: no such device or address
    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]

    assert_match "llama", pipe_output(bin/"llama", "--help").strip
```

Comment (chenrui333) [inline Formula/llama.rb:15]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Comment (antonmedv) [reply Formula/llama.rb]: Unfortunately no, as this is a TUI lib to test we need a filesystem and PTY to simulate user behavior.

**Comment by antonmedv** [comment]
I think, I fixed everything.

**Comment by antonmedv** [comment]
Okay, found the issue.

File: Formula/llama.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/llama.rb]: Will it fail reliably without a filesystem? Because a PTY can be simulated.

Comment (antonmedv) [reply Formula/llama.rb]: Yes, as llama is tui file manager.

Comment (antonmedv) [reply Formula/llama.rb]: I can write a test same as in nnn file manager: https://github.com/Homebrew/homebrew-core/blob/8f38abd94169fd2dbf7efe2cabb6eae079c6352e/Formula/nnn.rb

**Comment by antonmedv** [comment]
Now llama has test as in nnn/ranger tui.

**Comment by antonmedv** [comment]
Is there something else should I add?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@antonmedv, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**